bin/test_itemized

```diff
@@ -14,6 +14,11 @@ GOLDEN_FILE=etc/test_itemized.out
 rm -f $RESULTS_OUT $SEQUENCER_OUT $PUBBER_OUT 
 touch $RESULTS_OUT $SEQUENCER_OUT $PUBBER_OUT 
 
+#pubber/bin/build
```

Comment:
don't leave dead code

Reply:
Accidental, was meant to be I commented, thanks